tests/integration-tests/configs/isolated_regions.yaml

```diff
@@ -0,0 +1,76 @@
+{%- import 'common.jinja2' as common with context -%}
+{%- set REGIONS = ["us-isob-east-1"] -%}
```

Comment:
What about DCA (us-iso-*)? Sometimes the regions may not have feature parity, could it be beneficial to include DCA in this config as well?

Reply:
Currently we are only focused on us-isob-east-1.
We will extend the configuration in the next few weeks covering also us-iso-east-1